chrome-extension/content.js

```diff
@@ -101,23 +97,50 @@ async function createSidebar(content) {
   commentTextarea.setAttribute('placeholder', 'Note to self');
   form.appendChild(commentTextarea);
 
-  // Create select dropdown with options fetched from categories db
-  const dropdownLabel = customCreateElement('label', ['label', 'mt-2'], 'Add Clipping to a Category?');
-  dropdownLabel.htmlFor = 'categoryDropdown';
-  form.appendChild(dropdownLabel);
-  
   sidebar.appendChild(form);
 
-  const categoryDropdown = await getCategoryDropdown(currentUserId);
-  categoryDropdown.id = 'categoryDropdown';
-  form.appendChild(categoryDropdown);
+  const userEmailNote = customCreateElement('div', []);
+  let mongoId;
+
+  // Get logged-in user info
+  chrome.extension.sendMessage({}, async (userInfo) => {
+    const email = userInfo.email;
+    const googleId = userInfo.id;
+
+    if (googleId) {
+      try {
+        const response = await fetch(serverUrl + '/user/getUserId/' + googleId);
+        mongoId = await response.json();
+
+        // Create select dropdown with options fetched from categories db
+        const dropdownLabel = customCreateElement('label', ['label', 'mt-2'], 'Add Clipping to a Category?');
+        dropdownLabel.htmlFor = 'categoryDropdown';
+        form.appendChild(dropdownLabel);
+
+        const categoryDropdown = await getCategoryDropdown(mongoId);
+        categoryDropdown.id = 'categoryDropdown';
+        form.appendChild(categoryDropdown);
+
+        userEmailNote.innerHTML = 'Signed in as ' + email;
+      } catch (error) {
+        // TODO: do something if there's an error
```

Comment:
would at least printing error stack to console help future debugging?

Reply:
yes, will do for now